pydoctor/epydoc2stan.py

```diff
@@ -50,10 +55,12 @@ def get_parser(obj):
             formatname, e.__class__.__name__, e)
         obj.system.msg('epydoc2stan', msg, thresh=-1, once=True)
         mod = pydoctor.epydoc.markup.plaintext
-    return mod.parse_docstring
+    return getattr(mod, 'parse_docstring') # type: ignore[no-any-return]
```

Comment:
The `getattr` is a way to get around a mypy warning about arbitrary modules not having a `parse_docstring` attribute. However, since we need an ignore comment anyway, maybe it would be easier to silence the original warning instead.